chord_metadata_service/mohpackets/tests/endpoints/factories.py

```diff
@@ -73,16 +73,20 @@ class Meta:
         ),
         no_declaration=None,
     )
-    date_of_birth = {
-        "month_interval": random.randint(0, 100),
-        "day_interval": random.randint(0, 300),
-    }
+    date_of_birth = factory.LazyFunction(
+        lambda: {
+            "month_interval": random.randint(0, 1000),
```

Comment:
Wouldn't this make the month_interval and day_interval not line up? E.g. you could have a day_interval of 0 and a month_interval of 1000.

Reply:
You are correct, but I'm not testing the date logic here. I agree that a more realistic generator would be better, but these mock data serve my purpose for now.

Reply:
Sounds good to me -- maybe document this discrepancy in the Note: at the top of this file. Otherwise it looks good :heavy_check_mark:

Reply:
done